plugins/kotlin/jvm-debugger/core/src/org/jetbrains/kotlin/idea/debugger/core/stackFrame/KotlinStackFrame.kt

```diff
@@ -61,8 +60,12 @@ open class KotlinStackFrame(
         if (!removeSyntheticThisObject(evaluationContext, children, existingVariables) && thisVariables.isNotEmpty()) {
             remapThisObjectForOuterThis(evaluationContext, children, existingVariables)
         }
-
         children.add(evaluationContext, thisVariables, otherVariables)
+        KotlinStackFrameValueContributor.EP.extensions.forEach { contributor ->
```

Comment:
We usually use the `for {}` loop instead (unless a callable reference is passed).